src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -215,25 +219,28 @@ public static void ItemRender(RenderHandItemEvent.Pre event) {
         }
     }
 
+    private static final Set<ResourceKey<Level>> worldsWithoutRainOrThunder = Set.of(
+            Methods.moon,
+            Methods.moon_orbit,
```

Comment:
A `Set<>` is a special kind of `List<>` that doesn't care about the order of things inside it, which makes it faster than a normal `List<>` if we don't care about ordering either. The other difference is that a `Set<>` can't have more than one of the same item inside it, this also makes it faster than a normal list.

Reply:
i like this idea, can you do it pls for all things that use the planet list and pls move it in Methods

Reply:
Yes, once you merge or apply the changes yourself I'll do another PR that moves it and applies it to all things that use planets. This is to avoid merge conflict headaches with GitHub

Reply:
yep do a new pr pls